benchmark/README.md

```diff
@@ -1,6 +1,6 @@
 # Benchmarks
 
-created for SpeedyWeather.jl v0.9.0 on Tue, 02 Apr 2024 14:05:51. 
+created for SpeedyWeather.jl v0.10.0 on Wed, 25 Sep 2024 18:36:59. 
```

Comment:
Should we already change the version to 0.11 here?

Reply:
Yeah, makes sense. It's a major change, and some of it is also user facing (the output)